src/newbank/server/NewBankClientHandler.java

```diff
@@ -65,4 +65,25 @@ public void run() {
         }
     }
 
-}
+    public static String transfer(String request) {
+        try {
+            out.println("Which account would you like to transfer from?");
+            String fromAccount = in.readLine();
+            out.println("Which account would you like to transfer to?");
+            String toAccount = in.readLine();
+            out.println("How much would you like to transfer?");
+            String moveAmount = in.readLine();
+            double d = Double.parseDouble(moveAmount);
+            out.println(d);
+            Double acc = Account.getOpeningBalance();
+            if (d> 0 && d <= acc) {
+                return "success!";
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return "Fail";
+    }
```

Comment:
As a consequence of incorrectly using class variables in the other parts, this will not work. To transfer, we need to work with two instances of `Account`, not the class itself.